extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -132,38 +133,164 @@
 );
 
 /** Message documentation (Message documentation)
- * @author PtM
- * @author Siebrand
+ * @author Damian <damian@wikia-inc.com>
  */
 $messages['qqq'] = array(
-	'userprofilepage-question-caption' => 'Parameters:
+	'userprofilepage-edit-modal-header' => 'Header for edit profile modal on user profile page.',
+	'userprofilepage-question-caption' => 'Currently not in use!
+	Parameters:
 * $1 is the number of the current question
 * $2 is the total number of questions in the interview.',
-	'userprofilepage-question-list-title' => 'Parameters:
+	'userprofilepage-lightbox-interview-cancel' => "Currently not in use!",
```

Comment:
Unused messages are going to be cleaned up in separate ticket: https://wikia-inc.atlassian.net/browse/SOC-416